scripts/rate.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+### rate -- list all rates or compute specific ones
+### Usage:
+### ./rate.sh list all rates
+### ./rate.sh <entry> return the computed rate
```

Comment:
this comment is confusing since the make command requires `pct=<entry>`

Reply:
yeah, this refers to the script usage as it is, the `pct` part could be specified as error message on the sanity checks